includes/PHPMailer/PHPMailer.php

```diff
@@ -0,0 +1,5126 @@
+<?php
```

Comment:
I think there is no need to embed the library, because for example, I would like to install it from Debian: https://packages.debian.org/fr/sid/libphp-phpmailer

Your code would still work since it's in the global php include of the machine
Could you make this more modular to not require external libs ?

Reply:
Yup, but most probably the autoloader code should be moved to `autoload.php` and support a folder named vendor for example
that would allow composer users to install libs for the project